storage/myisam/mi_cache.c

```diff
@@ -1,4 +1,5 @@
 /* Copyright (c) 2000, 2015, Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2018, Percona and/or its affiliates. All rights reserved.
```

Comment:
Should (C) be for Percona, MariaDB, or both here?

Reply:
Fixed. Put both.